test/gurka/test_route_incidents.cc

```diff
@@ -83,57 +84,61 @@ void check_incident_locations(const valhalla::Api& api,
 
     // also check that incidents are in the right order
     uint32_t last_shape_index = 0;
-    for (const auto& i : leg.incidents()) {
-      ASSERT_GE(i.begin_shape_index(), last_shape_index) << " leg incidents were out of order";
-      last_shape_index = i.begin_shape_index();
+    for (const auto& incident : leg.incidents()) {
+      ASSERT_GE(incident.begin_shape_index(), last_shape_index) << " leg incidents were out of order";
+      last_shape_index = incident.begin_shape_index();
     }
     incident_count += leg.incidents_size();
   }
 
   // check all the incidents are in the right place in the route
-  for (const auto& i : locations) {
-    const auto& leg = api.trip().routes(0).legs(i.leg_index);
+  for (const auto& location : locations) {
+    const auto& leg = api.trip().routes(0).legs(location.leg_index);
     // find the incident
-    const valhalla::TripLeg::Incident* incident = nullptr;
+    const valhalla::TripLeg::ValhallaIncident* incident = nullptr;
```

Comment:
should have named this encapsulation just `Incident` since its namespaced to `TripLeg`